README.md

```diff
@@ -1,2 +1,14 @@
 # hw01
-Repository for homework 1
+Homework 1 from Donghyun Kang
+
+<img src="https://raw.githubusercontent.com/nerdizzyz/hw01/master/KANG_DONGHYUN.jpg">
+
+# Donghyun Kang's Biography
+
+My name is Donghyun Kang, a 1st-year Ph.D. student in the department of sociology. I was born in **Seoul**, South Korea on Sep 25, 1989. I did my B.A. at Seoul National University majoring in Business Administration and shifted to sociology for master's degree at the same university. For my master's thesis, I analyzed how the tweets regarding the two social movement events - *the LGBT group's sitting in the City Hall of Seoul* and *the protests of laid-off workers of Ssangyong Motor* - both of which happened in South Korea in 2014. I completed my master's degree in Feb 2016 and Before coming to Chicago in Aug 2017, I worked as a research associate at Social Network Computing Center at Seoul National University for one and a half of years. I'm currently interested in how and when the new academic disciplines emerge. More specifically, I hope to figure out the societal conditions that cause *interdisciplinary studies* to emerge. 
+
+* Research interests
+  Sociology of Knowledge; Computational Social Science; Economic Sociology; Social Organization
```

Comment:
You already fulfill the requirements for the assignment, but could have put these research interests in a list if you wanted